frontend/app/views/main.scala.html

```diff
@@ -44,6 +44,9 @@
         <meta property="fb:app_id" content="@Config.facebookAppId"/>
         <meta name="twitter:site" content="@@@Social.twitterUsername"/>
         <meta name="twitter:card" content="summary"/>
+        @for(pageImage <- pageInfo.image) {
```

Comment:
just checked, image is an Option, so we won't get a duplicate 👍  https://github.com/guardian/membership-frontend/blob/53a2cb56c55c6a63d15f773cbf71f58fc8aef02f/frontend/app/views/support/PageInfo.scala#L14